print("action worked")
raise Exception("action failed")